### PR TITLE
README: Explain the OpenSSL::Query requirements

### PR DESCRIPTION
**Pull request by levitte**
... also, reinstate git.  We do depend on it quite a lot.

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

**Review by t8m — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -66,7 +66,9 @@ a number of programs to be installed:
 
 -   [Template Toolkit].  The Debian package is `libtemplate-perl`
 -   [Pandoc].  The Debian package is `pandoc`
--   `python3`, `wget` and `OpenSSL::Query` perl module.
+-   [OpenSSL::Query], a perl module that you install in normal perly way.
```
Comment (mspncp) [inline README.md]: "...install in normal perly way" may be a completely obvious (and hence superfluous) or a completely unhelpful description of how to do it, depending on whether you are a Perl Guru or not. 😉

Comment (paulidale) [reply README.md]: Agreed, I almost commented about this too.  A pointer to the readme perchance?

Comment (levitte) [reply README.md]: Did you bother to follow the link?

Comment (paulidale) [reply README.md]: I did see the link.  I still think this could be improved.

Comment (levitte) [reply README.md]: See the fixup commit I just pushed

**Review by mspncp — APPROVED** [review]

**Comment by levitte** [comment]
@t8m, @paulidale, re-affirm your approvals?

**Review by paulidale — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by levitte** [comment]
Merged

288e14028659fc779f7780bc3f9403427a661464 README: Explain the OpenSSL::Query requirements

**Pull request closed without merging**